module-2/backend/week_one.md

```diff
@@ -5,30 +5,125 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+        Post - submits data to a specified resource
+        Get - retrieves data from a specified resource
+        Put - updates a resource completely through a specified resource
+        Delete - deletes a specified resource
+        I believe the fifth verb is 'Patch' and it does the same thing as 'Put'.
+        
 2. What is Sinatra?
+
+        A DSL that enables you to easily create web applications in Ruby.
+        
 4. What is MVC?
+
+        It's an acronym for Model, View, Controller. This is a software design pattern for implementing user interfaces on computers.
+        
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+        It simplfifies desicion making when creating an application. There is no need to waste time on how you will name your paths because that convention is already implemented in Sinatra. You can of course deviate from this convention but it will require you to write more code.
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+        All of the corresponding model's attributes and methods.

```

Comment:
This is true, but how do you get the model into the view? Generally, in order to maintain a separation of responsibilities we *collect* our data in the controller. Then you can use an **instance variable** to pass it to the view (as you're doing in question 7 below).